Terminal.Gui/Views/TextView.cs

```diff
@@ -142,6 +142,11 @@ public override string ToString ()
 		/// </summary>
 		public int Count => lines.Count;
 
+		/// <summary>
+		/// Gets or sets if null terminated in a buffer array is considered as part of the <see cref="lines"/>.
+		/// </summary>
+		public bool AllowsNullTerminated { get; set; }
```

Comment:
It doesn't matter if the `\0` is at the end of the line or not, right? So this has nothing to do with 'NullTerminated', but just `Null`.

Also, I believe the logic you implemented means that if this member is `true` any `\0` characters in the input stream will be preserved (and thus displayed in the editor). If `false` they will be ignored.

Thus, I believe this member should be named `AllowsNulls` instead. 

If I'm correct please rename it to `IgnoreNulls` and update the help text to something like 

> 'Gets or sets if '\0' characters will be preserved in the input stream by the <see cref="LoadStream(Stream stream)"/> method. If set to <see langref="false"/> (the default) any `\0` characters in the input stream will be preserved (and rendered in the TextView). If set to <see langref="true"/> `\0` characters will be stripped from the input stream."

And what will the editor show when `\0` characters are preserved in the input stream? Will it show a U+2400 (␀)?

I may be completely misunderstanding this though, so please correct me if I'm wrong!

Reply:
@tig I really I'm confused about this. This issue come up because I was getting wrong value on the unit test because of this statement:
```cs
var text = "Testing";
byte [] buff = System.Text.Encoding.Unicode.GetBytes (text);
```
`buff` returned an array length of 14, the double of 7 that is the length if the `text` variable. So, the `GetBytes` method was returning a null terminated for each character and not only at the end of the line. Maybe was my mistake of the using of the `GetBytes` method. If you think it's better not implementing this property at all I have no problem by remove it.
If you want to preserve I'm now with a doubt if I should named `AllowsNulls` or `IgnoreNulls`.

> And what will the editor show when \0 characters are preserved in the input stream? Will it show a U+2400 (␀)?

Yes it will.

Reply:
I think you should simplify this fix to fix the crash in the most simple way possible.

Reply:
You are meaning I should discard this `AllowsNullTerminated` or whatever property, right?

Reply:
Done @tig. I removed the `AllowsNullTerminated` property then.